### PR TITLE
Improve styling of case study on mobile and tablet

### PR DESCRIPTION
**Pull request by eadpearce**
previous:
Screenshot 2019-05-07 at 13 05 46

new:
Screenshot 2019-05-03 at 16 32 45

### CODE REVIEW

**Comment by Kenneth-Siu** [comment]
Sorry to be.... that person...

These case studies also turn up in other pages like the country guides in domestic. Would you be able to put this into directory-components for further reuse?

**Comment by Kenneth-Siu** [comment]
It also occurs to me that I'm the person to blame for it not being in directory-components in the first place 😅

File: core/sass/partials/_utils.scss [hunk 1]
```diff
@@ -82,3 +82,13 @@
...
+
+.only-display-tablet {
+  display: none;
+}
+
+@media (max-width: 768px) {
+  .only-display-tablet {
+    display: block;
+  }
+}
```
Comment (Kenneth-Siu) [inline core/sass/partials/_utils.scss]: Just a small thing: for the purposes of being mobile-first, would it make sense to write it the other way around? i.e.

```
.only-display-tablet {
  display: block;
}

 @media (min-width: 769px) {
  .only-display-tablet {
    display: none;
  }
}
```

File: core/templates/core/sector_page.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 <section id="breadcrumb-section">
   <div class="container">
-    <nav class="breadcrumbs" aria-label="breadcrumb" role="navigation">
+    <nav class="breadcrumbs" aria-label="Breadcrumbs">
```
Comment (richtier) [inline core/templates/core/sector_page.html]: if you're feeling fancy you could use the new breadcrumbs component :dancer:

Comment (eadpearce) [reply core/templates/core/sector_page.html]: haha i was going to but that's in another PR :D

File: core/sass/partials/_utils.scss (hunk 1 above)
Comment (eadpearce) [reply core/sass/partials/_utils.scss]: yeah we need to get some consistency with this in the style library too... will change this

**Comment by eadpearce** [comment]
going to plonk this in directory-components instead as @Kenneth-Siu said it's also used on domestic markets pages

**Pull request closed without merging**